tests/suite/foundations/decimal.typ

```diff
@@ -0,0 +1,51 @@
+--- decimal-constructor ---
+#test(decimal(10), decimal("10.0"))
+#test(decimal("\u{2212}7654.321"), decimal("-7654.321"))
+#test(type(decimal(10)), decimal)
+
+--- decimal-constructor-bad-type ---
+// Error: 10-17 expected integer or string, found type
+#decimal(decimal)
+
+--- decimal-constructor-bad-value ---
+// Error: 10-17 invalid decimal: 1.2.3
+#decimal("1.2.3")
+
+--- decimal-repr ---
+// Test the `repr` function with decimals.
+#set page(width: auto)
```

Comment:
This test doesn't need a reference image. Better just use `test` assertions.

Reply:
Good one. We may want to update the float and int tests as well (I copied this from float).

Reply:
Yes there are various tests across the test suite that could be migrated away from images. I didn't have the time to rewrite them all, but I try to avoid unnecessary images for new ones.